RecoEgamma/EgammaElectronAlgos/src/ElectronSeedGenerator.cc

```diff
@@ -148,8 +148,8 @@ ElectronSeedGenerator::ElectronSeedGenerator(const edm::ParameterSet &pset, cons
       // new B/F configurables
       deltaPhi2B_(pset.getParameter<double>("DeltaPhi2B")),
       deltaPhi2F_(pset.getParameter<double>("DeltaPhi2F")),
-      phiMin2B_(pset.getParameter<double>("PhiMin2B")),
-      phiMin2F_(pset.getParameter<double>("PhiMin2F")),
+      phiMin2B_(-pset.getParameter<double>("PhiMax2B")),
```

Comment:
Are these boundaries always supposed to be symmetric?
(Same question for the other parameter changes below)

Reply:
I think there is really an extremely small chance that we will ever consider electrons and positrons to be anything more than charge-inverted in the CMS reconstruction, so I really don't get why it was at all possible to configure this in a non-symmetric way. I think this was a clear case of over-configurabilty. Maybe I can argue that this does not matter so much at this point anyway, because as @jainshilpi said, it it very likely that the `ElectronSeedProducer` will be hopefully replaced by the `ElectronNHitSeedProducer` in the near future anyway? That's why I don't have a strong preference, but leaving it as it is in the PR right now is of course easier.

Reply:
It makes sense, just wanted to be sure there were not possible use cases for the contrary, since it was originally coded as such,
Taking also into account the intention of moving eventually to ElectronNHitSeedProducer, ok for moving towards the simpler version.